docs/src/userguide/glossary.rst

```diff
@@ -0,0 +1,194 @@
+.. _glossary:
+
+Glossary
+=============
+
+.. glossary::
+
+   Cartopy
+        A python package for producing maps, and other geospatial data.
+        Allows plotting on these maps, over a range of projections.
+
+        | **Related:** :term:`Matplotlib`
+        | **More information:** `CartoPy Site <https://scitools.org.uk/cartopy/docs/latest/>`_
+        |
+
+   Coordinates
+        A container for data points, comes in three main flavours.
+
+        - Dimensional Coordinates -
+            A coordinate that represents a singular data dimension.
+            These are organised in ascending order of dimension. Can only
+            contain numerical data.
```

Comment:
```suggestion
            Can only contain numerical values, in a sorted order (ascending or descending).
```
Otherwise I think this sounds like : "the dim-coords themselves are organised in dimension order"